pkg/server/storage_api/engine_test.go

```diff
@@ -19,25 +20,36 @@ import (
 	"github.com/cockroachdb/cockroach/pkg/util/leaktest"
 	"github.com/cockroachdb/cockroach/pkg/util/log"
 	"github.com/pkg/errors"
+	"github.com/stretchr/testify/require"
 )
 
 // TestStatusEngineStatsJson ensures that the output response for the engine
 // stats contains the required fields.
 func TestStatusEngineStatsJson(t *testing.T) {
 	defer leaktest.AfterTest(t)()
 	defer log.Scope(t).Close(t)
+	ctx := context.Background()
 
 	dir, cleanupFn := testutils.TempDir(t)
 	defer cleanupFn()
 
 	srv := serverutils.StartServerOnly(t, base.TestServerArgs{
-		DefaultTestTenant: base.TestIsForStuffThatShouldWorkWithSecondaryTenantsButDoesntYet(110020),
+		DefaultTestTenant: base.TestTenantProbabilisticOnly,
 
 		StoreSpecs: []base.StoreSpec{{
 			Path: dir,
 		}},
 	})
-	defer srv.Stopper().Stop(context.Background())
+	defer srv.Stopper().Stop(ctx)
+
+	if srv.DeploymentMode().IsExternal() {
+		// explicitly enabling CanViewNodeInfo capability for the secondary/application tenant
+		// when in external process mode, as shared process mode already has all capabilities.
```

Comment:
Please fix the first letter of the first word of sentence to uppercase. 

Suggestion: I think at this point we don't need these two lines of comment, it's sort of a common knowledge I guess.